random_forest_function.ipynb

```diff
@@ -0,0 +1,344 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "09a88133",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from path import Path\n",
+    "import pandas as pd\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import confusion_matrix, accuracy_score, classification_report\n",
+    "from matplotlib import pyplot as plt\n",
+    "import numpy as np\n",
+    "import sqlite3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fda8a54b",
+   "metadata": {},
+   "source": [
+    "## Import Data from Database"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6da076dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#change room occupancy count to occupancy in Data 1\n",
+    "\n",
+    "con = sqlite3.connect('sensor-telemetry.sqlite')\n",
```

Comment:
For consistency between the ETL and ML stages, we should stick with SQLAlchemy.
```
from sqlalchemy import create_engine
con = create_engine("sqlite:///sensor-telemetry.sqlite")
```

Reply:
Done! Good catch!